samples/snippets/page_management.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2021, Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.cloud.dialogflowcx_v3 import PagesAsyncClient
+from google.cloud.dialogflowcx_v3.types.page import (
+    CreatePageRequest,
+    DeletePageRequest,
+    ListPagesRequest,
+    Page,
+)
+
+
+# [START dialogflow_cx_create_page]
+async def create_page(project_id, agent_id, flow_id, location, displayName):
+    pages_client = PagesAsyncClient()
+
+    page = Page()
+    page.display_name = displayName
+
+    request = CreatePageRequest()
+    request.parent = (
+        "projects/"
```

Comment:
Is there a reason we are concatenating a string? Would an f string perhaps be a bit fewer lines and easy to read?  f"projects/{project_id}/locations/{location}..."